packages/cascara/src/components/Dashboard/widgets/WidgetStats.js

```diff
@@ -18,10 +17,12 @@ const WidgetStats = ({ data, ...rest }) => {
   const dataState = getDataState(data);
 
   return (
-    <Widget {...rest} {...dataState} className={styles.Stats} height='auto'>
```

Comment:
The Columns component works so fine here. Just the appearance was altered:
![image](https://user-images.githubusercontent.com/87443054/151107667-e85803ec-f649-42ad-aa08-454a87db4e5e.png)
![image](https://user-images.githubusercontent.com/87443054/151107713-7e15a97f-1db1-4298-87d9-367f9d3d4475.png)
Should we keep `styles.Stats` or should we work on the class of the Stat component?

Reply:
We can take a look. If I recall, there is a Stat specific override that is making the grey background go away on these that we will want to keep. But we can also possibly simplify the styles.